### DISCUSSION TITLE
cannot assign elasticsearch field to jira_labels

### DISCUSSION THREAD

**Discussion by sucremad** [jertel/elastalert2#1356] [Q&A] [answered]
Hi,

I want to take the labels in the field signal.rule.tags to jira_labels. Isn't it possible? I tried the field in many forms but couldn't make it. such as:

`jira_labels: signal.rule.tags`
`jira_labels: signal[rule][tags]`
`jira_labels: {signal[rule][tags]}`
`jira_labels: "{signal[rule][tags]}"`
`jira_labels: {signal.rule.tags}`
`jira_labels: "{signal.rule.tags}"`

How should I write?

**Comment by jertel** [comment, marked as answer]
Unfortunately that `jira_labels` field is currently only capable of handling static values. If you are up to submitting a PR to make it perform a match field lookup that would be a good improvement.
